### ISSUE TITLE
Example with Blazor and Google authentication

### ISSUE THREAD

**Issue by jkone27** [passwordless-lib/fido2-net-lib#337]
Could you provide a toy/example app with Blazor aspnetcore app and google authentication?

**Comment by abergs** [comment]
Good idea. When I have time I hope to get around to improve examples and a blazor one is certainly on my todo.

**Comment by Regenhardt** [comment]
I'm actually working on using this in Blazor WASM right now, maybe I can make a PR once I have it actually functional.

**Comment by abergs** [comment]
Appreciated @Regenhardt

**Comment by Regenhardt** [comment]
The Blazor WASM part of this issue is included in #379. 

Not sure about google authentication, I feel like that is out of scope for this library as the point is that google provides the authentication and, if I understand those concepts correctly, FIDO is for authentication ("Who are you?", check via password/biometrics/SMS/E-Mail), not authorization ("What are you allowed to do?", checked via JWT or other tokens).  
The google part is however covered in other places already, like https://code-maze.com/google-authentication-in-blazor-webassembly-hosted-applications/, which you could try to reimplement.

**Comment by aseigler** [comment]
Resolved by #379
